### PR TITLE
Use stdout.isTTY instead of tty.isatty

### LINKED ISSUE #100: Use process.stdout.isTTY instead of node:tty
> The preferred method of determining whether Node.js is being run within a TTY context is to check that the value of the process.stdout.isTTY property is true:
>
> ```sh
> $ node -p -e "Boolean(process.stdout.isTTY)"
> true
> $ node -p -e "Boolean(process.stdout.isTTY)" | cat
> false
> ```
https://nodejs.org/api/tty.html#tty

This would also remove the dependency on the `node:tty` module and make it "more easier" for use in the browser. i.e. NO TRANSPILATION required

### PR DESCRIPTION
**Pull request by iamandrewluca**
Closes #100

### CODE REVIEW

**Comment by iamandrewluca** [comment]
Hey, @jorgebucaran, can you please take a look at this PR?! 👀

**Comment by jorgebucaran** [comment]
Do you think we can fix the failing tests? 🤔

**Comment by iamandrewluca** [comment]
I will take a look, but the failing tests seem to not be related to this PR 🤔 
I might open another PR to fix the tests.

**Comment by iamandrewluca** [comment]
I tried locally with different versions of node 10/12/14, and it seems the problem can be in the incompatibility of the libraries.

For example, we try to run rollup on node 10, but rollup minimum version is 18

**Comment by jorgebucaran** [comment]
Thank you! Yes, we don't really want to do that, so will need to update our infra a bit.

**Comment by iamandrewluca** [comment]
Hey, @jorgebucaran pushed a commit here to test if building with Rollup v2 works.
Can you please approve the workflows?

File: package.json [hunk 1]
```diff
@@ -28,13 +28,14 @@
...
     "test": "c8 twist tests/*.js",
-    "build": "npx rollup --format cjs --input index.js --file index.cjs",
+    "build": "npx rollup --external tty --format cjs --input index.js --file index.cjs",
     "deploy": "npm test && git commit --all --message $tag && git tag --sign $tag --message $tag && git push && git push --tags",
     "release": "tag=$npm_package_version npm run deploy && npm publish --access public",
     "prepare": "npm run build"
   },
   "devDependencies": {
     "c8": "*",
+    "rollup": "^2.79.1",
```
Comment (jorgebucaran) [inline package.json:38]: We don't need to list rollup if we're using npx, do we?

Comment (iamandrewluca) [reply package.json:38]: npx pulls the package from the registry if it is not found locally.
We can run it directly for smaller package download sizes.

I'm pushing another change.

**Pull request closed without merging**